pkgs/applications/graphics/ahoviewer/default.nix

```diff
@@ -37,7 +37,9 @@ stdenv.mkDerivation rec {
     description = "A GTK2 image viewer, manga reader, and booru browser";
     maintainers = with maintainers; [ skrzyp xzfc ];
     license = licenses.mit;
-    platforms = platforms.allBut [ "darwin" "cygwin" ];
+    # Unintentionally not working on Darwin:
+    # https://github.com/ahodesuka/ahoviewer/issues/63
```

Comment:
That's a dead link ;)

Reply:
https://github.com/ahodesuka/ahoviewer/issues/62 wow I incremented it somehow?